packages/framer-motion-3d/src/utils/use-time.ts

```diff
@@ -4,7 +4,7 @@ import { useContext } from "react"
 
 export function useTime() {
     const time = useMotionValue(0)
-    const { isStatic } = useContext(MotionConfigContext)
+    const isStatic = useContext(MotionConfigContext)["isStatic"] // Internal API
```

Comment:
I think this is the only real hack I had to do to dip into the internals without too much trickery.